### ISSUE TITLE
fpga_common: Remove red vs green status LED 

### ISSUE THREAD

**Issue by walliv** [CESNET/ndk-core#1]
This disctinction makes the code hard to read. First, not all of the cards have LEDs of these two colors, some have single color, some have three colors. Second, the colors do not have any representation in general (green for success, red for failure). Both are just notification LEDs. 

Make only one vector of status LEDs regardless of their color and split them eventually in fpga.vhd top-level entity.

**Comment by jakubcabal** [comment]
I agree with this suggestion, but I would go further... I would make pre-defined status leds with fixed meaning (PCIe link up, heartbeat and already existing eth link and activity) and for that there would be something like user app leds connected directly to the application core.
